leap_hand/README.md

```diff
@@ -5,30 +5,68 @@
+This package contains a simplified robot description (MJCF) of the (left and right) [LEAP Hand](https://leaphand.com/) developed by Carnegie Mellon University. It is derived from the [publicly available URDF description](https://github.com/dexsuite/dex-urdf/tree/main/robots/hands/leap_hand).
 
 <p float="left">
-  <img src="leap_hand_left.png" width="400">
-  <img src="leap_hand_right.png" width="400">
+  <img src="left_hand.png" width="400">
+  <img src="right_hand.png" width="400">
 </p>
 
 ## URDF → MJCF derivation steps
 
-1. Added `<mujoco> <compiler discardvisual="false" strippath="false" fusestatic="false" balanceinertia="true"/> </mujoco>` to the URDF's
-   `<robot>` clause in order to preserve visual geometries.
-2. Removed `package://` shortcuts in the URDF.
-3. Loaded the URDF into MuJoCo and saved a corresponding MJCF.
-4. Manually edited the MJCF to extract common properties into the `<default>` section.
-5. Added `exclude` clause to prevent collisions between the palm and the base of each finger as well as the mcp and dip joint of each finger.
-6. Added position controlled actuators.
-7. Added `impratio=10` and `cone=elliptic` for better noslip.
-8. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
-
-Extra steps taken for the left hand:
-
-- Renamed `palm_lower_left` to `palm_lower`.
-- Replaced `thumb_left_temp_base` mesh with `pip` to match the right hand.
-- Reordered the joints of the thumb to match that of the right hand.
+### Right Hand
+1. Start with the right-handed LEAP hand URDF from [dexsuite (hash: 2ee2f70cfd549857f56ffe13c61fa939f4cb50ba)](https://github.com/dexsuite/dex-urdf/tree/main/robots/hands/leap_hand)
+2. Run it through drake's fix_inertia tool (v. 1.30.0)
+3. Add the following mujoco block to the urdf under `<robot>`:
+    ```
+    <mujoco>
+      <compiler balanceinertia="true" discardvisual="false" fusestatic="false"/>
+    </mujoco>
+    ```
+4. Open the file in the mujoco viewer using
+    ```
+    python -m mujoco.viewer --mjcf=<name_of_inertia_fixed_urdf>.urdf
+    ```
+5. Save the xml out of the viewer by clicking the "Save xml" button. From now on, modifications are referring to the saved xml.
+6. Delete the auxiliary body "base" created by not fusing static bodies.
+7. Rename bodies and joints according to the following:
+    - palm_lower -> palm
+    - mcp_joint -> base
+    - pip -> proximal
+    - dip -> medial
+    - fingertip -> distal
+    - white_tip -> tip
+8. Give names to all geoms according to visual or collision status.
+9. Create default classes for joints and assign all joints to their appropriate class.
+10. Delete the actuatorfrcrange field for all joints.
```

Comment:
Any reason `actuatorfrcrange` was deleted?

Reply:
I don't 100% remember, but I think there were some weird interactions between the actuatorfrcrange and the position-controlled actuator when I was tuning the model a few months ago that were resolved when I removed this field. We could try adding it back, not sure the best way to test the model afterwards.